source/guide_synapse.rst

```diff
@@ -0,0 +1,412 @@
+.. author:: 927589452
+.. highlight:: console
+
+.. tag:: python
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/matrix-logo.svg
+      :align: center
+
+######
+Synapse
+######
+
+.. tag_list::
+
+Synapse_ is the reference implementation of a matrix server.
+Matrix_ is federated chat protocol aiming to replace xmpp.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :lab:`Postgresql <guide_postgresql>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://github.com/matrix-org/synapse/blob/master/LICENSE
+
+Prerequisites
+=============
+
+You need a running :lab:`Postgresql <guide_postgresql>` database server, a dedicated user with a secure password and database for synape.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ createuser synapse -P
+  Enter password for new role:
+  Enter it again:
+  [isabell@stardust ~]$ createdb \
+    --encoding=UTF8 \
+    --owner="synapse" \
+    --template=template0 \
+    synapse
+  [isabell@stardust ~]$
+
+Installation
+============
+
+We will install synapse using pip, which makes it quite easy,
+but we start with creating a virtual environment for synapse to run in.
+
+.. code-block:: console
+  :emphasize-lines: 1,2,8,18,27,37
+
+  [isabell@stardust ~]$ mkdir -p ~/synapse
+  [isabell@stardust ~]$ virtualenv -p python3.6 ~/synapse/env
```

Comment:
What is your opinion on installing Synapse with `pip install --user` instead? That way the installation and service setup is easier to handle and less prone to mistakes.

Reply:
@927589452 what do you thinkk?

Reply:
If you're not interested in changing the guide, but do generally not oppose the `--user` way of doing things, please say so. I'm happy to do it :grin:

Reply:
I can do it; the original source i went from used the venv and the official guide does so too https://github.com/matrix-org/synapse/blob/master/INSTALL.md